plugin.video.vrt.nu/addon.xml

```diff
@@ -48,39 +48,56 @@ Met deze Kodi add-on kan je alle video streams van VRT NU bekijken.
         <platform>all</platform>
         <license>GNU General Public License, v3</license>
         <news>
+v2.2.1 (2019-08-16)
```

Comment:
Keep this only for the current version

Reply:
@dagwieers Please keep this in mind for the next version.

Reply:
We like to keep more than one entry in the changelog, and especially in this case where v2.2.0 was not released due to this review process.

What is the reason for limiting the "news" entry to 1500 characters and just one entry?